test.json

```diff
@@ -0,0 +1 @@
+{"obj(ect)?/[\\d-]+/": {"endpoint": "http://localhost/api", "processor": "app.tests.test_config.DummyProcessor"}}
```

Comment:
Hmm, why is this necessary? Did the tests not work before?

Reply:
I thought that file came from you, but I now see that it came from my commit. I've removed it.